connectors/cli.py

```diff
@@ -128,6 +127,28 @@ async def _start_service(actions, config, loop):
         return await multiservice.run()
 
 
+def get_event_loop(uvloop=False):
```

Comment:
Out of curiosity, why did `get_event_loop` have to be moved?

Reply:
Cause it had a "try/except" with `except` that does nothing. I've added a logger statement there and moved to the only place where the function is used